ui/docs/reference/core/worker-manager/worker-interaction.mdx

```diff
@@ -48,6 +48,16 @@ It is the responsibility of the worker to finish its work before the credentials
 
 Depending on the provider, `registerWorker` may be called multiple times for the same worker, or may only be called once.
 
+## Reregistration
+
+The worker may reregister itself to renew its credentials with the worker manager by calling
+[`workerManager.reregister`](/docs/reference/core/worker-manager/api#reregisterWorker) and
+supplying it with a one time usage `secret`. The secret value will have been either last configured
```

Comment:
"configured" here implies to me that it is a user-supplied value. Let's rephrase this to say that worker-manager generates these values.